README.md

```diff
@@ -30,3 +30,7 @@ If you're considering contributing, a working knowledge of the following would b
 
 <img src="https://raw.githubusercontent.com/Piwigo/Piwigo-Android/master/assets/screenshots/albums-phone.png" width="238" />
 <img src="https://raw.githubusercontent.com/Piwigo/Piwigo-Android/master/assets/screenshots/albums-tablet.png" width="600" />
+
+## Licensing
+The project is delivered under the GPL in version 3, or (at your option) any later version of that license. By submitting code to the project by patches, pull request or any other form the authors guarantee that it is either their own work and they accept to distribute their content under this license or the contribution is (re-)distributable under the terms of this license. If not specifically marked with a dedicated copyright notice, the copyright of the source code is transfered to to Piwigo project as far as permitted by applicable law. If necessary for legal reasons the source code author(s) shall be extracted from the meta data of the version control system.
```

Comment:
I think actually just adding the standard short license statement would be sufficient in the readme. The GPL already contains a section on copyright of contributed work.